site/templates/try.php

```diff
@@ -2,9 +2,15 @@
 
 <article>
 	<header class="mb-24">
-		<h1 class="h1">
-			Try Kirby
-		</h1>
+		<div class="max-w-xl">
+			<h1 class="h1 mb-6">
+				Try Kirby
+			</h1>
+
+			<p class="text-xl leading-snug color-gray-700">
+				Explore our online demo or download Kirby and try it out yourself to see how it fits your project. Ready to launch? <a class="color-black link" href="/buy">Buy a license.</a>
```

Comment:
Would keep it shorter:

Explore our online demo, or download Kirby and try it our yourself. Ready to launch? <a class="color-black link" href="/buy">Buy a license.</a>

i.e. without the